config_panel.toml

```diff
@@ -108,3 +108,15 @@ name = "Configuration"
         pattern.regexp = '^([0-9.]{7,15}|[0-9a-fA-F:]+)$'
         pattern.error = "Not an ip"
 
+        [main.hotspot.captive_portal]
+        ask = "Captive portal"
+        type = "boolean"
+        visible = "advanced"
+	help = "Activate the captive portal mode"
+
+        [main.hotspot.captive_portal_url]
+        ask = "Local captive portal URL"
+        type = "string"
+        visible = "advanced && captive_portal"
+	help = "Local URL on which redirect onto when the user mac address is not yet allowed"
```

Comment:
For some reason this field is understood as mandatory even when `advanced` / `captive_portal` ain't enabled though that sounds like a core issue rather than this app strictly speaking ...

Maybe it's not the case anymore in 12.x though because of the pydantinc refactoring  (the bug was only observed on 11.x)